recipes/kcov/all/conanfile.py

```diff
@@ -34,8 +35,9 @@ def source(self):
         os.rename(extracted_dir, self._source_subfolder)
 
     def _patch_sources(self):
-        for patch in self.conan_data["patches"][self.version]:
-            tools.patch(**patch)
+        if self.version == 38:
```

Comment:
Revert. The conandata.yml is place to list which patch should be applied to which version.